### PR TITLE
Move container build files into their own dir and introduce the scope script

### LINKED ISSUE #37: Properly Dockerize Scope
This work is mostly porting @tomwilkie's prerelease commits. End result is a working Docker container for Scope, plus coherent reproducible build instructions in the README.

### PR DESCRIPTION
**Pull request by tomwilkie**
Fixes #37

### CODE REVIEW

File: scope [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+usage() {
+    echo "$0 (launch|stop)"
+}
+
+SCOPE_IMAGE=weaveworks/scope
+CONTAINER_NAME=weavescope
```
Comment (peterbourgon) [inline scope]: `scope`?

**Comment by peterbourgon** [comment]
Other than that, :dango:

File: scope (hunk 1 above)
Comment (tomwilkie) [inline scope]: its weavedns and weave for the other containers.

**Comment by tomwilkie** [comment]
PTAL; I've added integration with weavedns.

**Comment by peterbourgon** [comment]
It's a shame that's so convoluted, but :dango:

**Pull request merged**